frontend/src/components/MapView/DateSelector/TimelineItems/index.tsx

```diff
@@ -37,56 +47,63 @@ const TimelineItems = memo(
 
     const { t } = useSafeTranslation();
 
+    const selectedLayerDateItems: DateItem[][] = useMemo(
```

Comment:
If we take the dateItems from the `selectedLayers` I don't think we need it right?

Reply:
I kept it to have 2 dimensions array of layers with dateItems